aws-xray/src/test/java/io/opentelemetry/contrib/awsxray/AwsMetricAttributeGeneratorTest.java

```diff
@@ -5,19 +5,32 @@
 
 package io.opentelemetry.contrib.awsxray;
```

Comment:
Did not have time to review tests, will rely on reviews from others to cover these.